### ISSUE TITLE
Should I watch this?

### ISSUE THREAD

**Issue by janat08** [pkdone/gke-mongodb-demo#1]
Are there any other settings you wish to implement like SSL or something. Also resource allocation doesn't claim to be automatic and so sounds unreasonable for some defaults to take place without a clue.

**Comment by pkdone** [comment]
TLS/SSL would be a reasonable addition. Demo is mainly just trying to be bare minimum that adds some level of being secure by default (i.e. 'auth' enabled) plus not use the https://github.com/cvallance/mongo-k8s-sidecar approach which is dangerous. More context is here: [http://pauldone.blogspot.co.uk/2017/06/deploying-mongodb-on-kubernetes-gke25.html](http://pauldone.blogspot.co.uk/2017/06/deploying-mongodb-on-kubernetes-gke25.html)

Basically I expect users to build their own environment but the demo at least provides an example building block that is more resilient and more secure than current examples out there.
